code/game/objects/items/weapons/twohanded.dm

```diff
@@ -272,3 +272,36 @@
 
 	force = 16
 	throwforce = 25
+
+/obj/item/binoculars
+	name = "binoculars"
+	desc = "Used for long-distance surveillance."
+	icon_state = "binoculars"
+	item_state = "syringe_kit"
+	gender = PLURAL
+	flags = FPRINT | TWOHANDABLE
+	slot_flags = SLOT_BELT
+	w_class = W_CLASS_SMALL
+	var/backup_view = 7
```

Comment:
That could also cause problems if their view changes while zoomed
Adding and subtracting is safer

I mean almost everything that changes your view already does so unsafely, but we should probably stop adding more things that do